reconcile/quay_base.py

```diff
@@ -16,10 +16,10 @@ def get_quay_api_store():
     store = {}
     for org_data in quay_orgs:
         name = org_data['name']
-        server_url = org_data.get('serverUrl')
+        api_url = org_data['instance']['url']
```

Comment:
The previous version of this doesn't assume `serverUrl` will exist. Where is it declared that either `instance` or `instance.url` will definitely exist?